### DISCUSSION TITLE
Migration from TinyMediaManager to MediaElch

### DISCUSSION THREAD

**Discussion by jberli** [Komet/MediaElch#1529] [General]
I was wondering if some people had successfully migrated from TinyMediaManager to MediaElch?
I have a collection of ~4000 films already scraped by hand using TMM and I want to migrate.
Does MediaElch keep the old nfo from TMM? Do I need to rescrape all?

Any tips about the migration is appreciated as well !

**Comment by bugwelle** [comment]
Hi, MediaElch uses existing NFO files. Only if you scrape a movie, MediaElch will change the NFO. I can't say whether you will loose any details. If there are XML tags that MediaElch does not support, they will most likely vanish once you scrape and save a movie.
